### ISSUE TITLE
Arithmetic exception running C++ vio_jsonl with OAK-D S2

### ISSUE THREAD

**Issue by RR-BenSteer** [SpectacularAI/sdk-examples#31]
Hello, 

I've built the **C++ OAK-D** examples by linking to the libs from the Linux_Ubuntu_x86-64 release (spectacularAI_depthaiPlugin_cpp_non-commercial_1.1.1.tar.gz), but I've had the following issue when running the _**vio_jsonl**_ example:

```
Stack trace (most recent call last):
#6    Object "[0xffffffffffffffff]", at 0xffffffffffffffff, in 
#5    Object "./vio_jsonl", at 0x55e1a8dd1e99, in _start
#4    Object "/lib/x86_64-linux-gnu/libc.so.6", at 0x7f3cd244cc86, in __libc_start_main
#3    Object "./vio_jsonl", at 0x55e1a8dd1fea, in main
#2    Object "/usr/local/lib/libspectacularAI_depthaiPlugin.so.1", at 0x7f3cd45765d5, in spectacularAI::daiPlugin::Pipeline::Pipeline(dai::Pipeline&, spectacularAI::daiPlugin::Configuration const&)
#1    Object "/usr/local/lib/libspectacularAI_depthaiPlugin.so.1", at 0x7f3cd4576963, in spectacularAI::daiPlugin::Pipeline::Pipeline(dai::Pipeline&, spectacularAI::daiPlugin::Configuration const&, std::function<void (std::shared_ptr<spectacularAI::mapping::MapperOutput const>)>)
#0    Object "/usr/local/lib/libspectacularAI_depthaiPlugin.so.1", at 0x7f3cd45796f2, in std::shared_ptr<dai::node::StereoDepth> dai::PipelineImpl::create<dai::node::StereoDepth>(std::shared_ptr<dai::PipelineImpl> const&)
Floating point exception (Integer divide by zero [0x7f3cd45796f2])
Floating point exception (core dumped)
```

I'm using Ubuntu 18.04.6 LTS and my architecture is indeed x86_64. I am using an **OAK-D S2** and I've tried both the _main_ branch and the _oak-d-s2_ branch of _sdk-examples_. I have the depthai-core libraries installed on my system already (built as shared objects).

Strangely, _vio_jsonl_ runs fine with the OAK-D S2 when I use the binary included with the Linux_Ubuntu_x86-64 release.

Any help with this would be appreciated, as we are trying out Spectacular AI for potential use in a real-time 3D reconstruction project :)

**Comment by Bercon** [comment]
Thank you for the bug report! We believe we've identified the cause and are aiming it to be fixed in the next release which is coming out next week

**Comment by RR-BenSteer** [comment]
Great, thanks for letting me know!

**Comment by Bercon** [comment]
The root cause for this was likely that the previous version was using v2.15.3 depthai-core while the docs instructed to use v2.16.0. We've now released 1.2.0 version of our SDK that uses v2.16.0. Please let me know if this fixes your issues: https://github.com/SpectacularAI/sdk/releases/tag/v1.2.0

**Comment by RR-BenSteer** [comment]
Hi, I tried building the OAK-D sdk example with the v.1.2.0 release and was able to run vio_jsonl successfully! I made sure depthai-core was v2.16.0. Thanks for your help.
